### PR TITLE
Troubleshooting section for getcwdu problem.

### PR DESCRIPTION
**Pull request by robrap**
FYI: @andy-armstrong

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -477,6 +477,31 @@ Or, you can run the following commands to clean up dangling images and volumes:
...
+Everyone who has seen this has gotten past it, but we don't have a surefire way
+to do so. Rebooting and restarting Docker do *not* seem to correct the issue. It
+may be an issue that is exacerbated by our use of sync (which typically speeds
+up the provisioning process on Mac), so you can try the following:
+
+.. code:: sh
+
+   # repeat the following until you get past the error.
+   make stop
+   make provision
```
Comment (mjfrey) [inline README.rst]: make dev.provision

File: README.rst [hunk 2]
```diff
@@ -477,6 +477,31 @@ Or, you can run the following commands to clean up dangling images and volumes:
...
+
+Once you get past the issue, you should be able to continue to use sync versions
+of the make targets.
```
Comment (mjfrey) [inline README.rst]: Not sure if this is true.  Once you provision without sync, you can not use sync.

**Review by mjfrey — COMMENTED** [review]
couple of comments.

File: README.rst (hunk 2 above)
Comment (robrap) [reply README.rst]: I'm pretty sure that sync can be used or not used at any time.  It just requires that sync is first run, which is what our make targets using sync use.
@clintonb @jbarciauskas @macdiesel: Can anyone confirm?

File: README.rst (hunk 1 above)
Comment (robrap) [reply README.rst]: I'll fix.  Thanks.

File: README.rst (hunk 2 above)
Comment (clintonb) [reply README.rst]: Correct. You can swap between using Docker Sync and native volumes at anytime.

**Review by mjfrey — APPROVED** [review]

**Review by macdiesel — APPROVED** [review]

**Pull request merged**